server/printing/SCEPrinter

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+#!/usr/bin/env python3
+# import printing_server.py
+import curses
+import argparse
+parser = argparse.ArgumentParser(description='Print the mock for 2D printer.')
+parser.add_argument('--mock-output', help="Displays Mock Output", )
+args = parser.parse_args()
+
+
```

Comment:
in order to render the number of copies each printer has printed over time, you may want to move your visual logic to be inside of a class. e.g.

```py
class ScePrinterVisualizer:
  left_copies = 0
  right_copies = 0
  def __init__(self):
    """
      this is a constructor, do whatever you want to initialize the class here.
    """
  def add_left_printer_copies(self, count):
    """
      take the parameter count and add it to self.left_copies
    """
  def add_right_printer_copies(self, count):

    """
      take the parameter count and add it to self.right_copies
    """
  def create_visual:
    """
      make use of self.right_copies and self.left_copies to render the ncurses visuals. here
    """      
```